README.md

```diff
@@ -1,8 +1,12 @@
-[![Documentation Status](https://readthedocs.org/projects/rocketpyalpha/badge/?version=latest)](https://rocketpyalpha.readthedocs.io/en/latest/?badge=latest)
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/giovaniceotto/rocketpy/blob/master/docs/notebooks/getting_started_colab.ipynb)
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/giovaniceotto/RocketPy/master?filepath=docs%2Fnotebooks%2Fgetting_started.ipynb)
-[![Downloads](https://pepy.tech/badge/rocketpyalpha)](https://pepy.tech/project/rocketpyalpha)
+[![PyPI](https://img.shields.io/pypi/v/rocketpy?color=g)](https://pypi.org/project/rocketpy/)
+[![Downloads](https://pepy.tech/badge/rocketpyalpha)](https://pepy.tech/project/rocketpy)
```

Comment:
Not 100% sure, but I think we had agreed on deleting this badge. Did you get a 2nd thought?

Reply:
From what I read in the meeting minutes; we had decided to substitute this badge if we could find a better alternative to showcase how many people are using RocketPy. Unfortunately, I couldn't find one. However, I can delete it everyone agrees.

Reply:
No problem, that's ok